core-feature-pack/src/main/resources/content/bin/domain.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+. ./common.sh
+
```

Comment:
This needs to happen after the `$DIRNAME` is configured and should be `. "$DIRNAME/common.sh`. Without that it can't be launched from any other directory than bin, e.g. `cd $JBOSS_HOME/bin && ./domain.sh`.